### PR TITLE
Lint fixes for golangci-lint 1.49.0

### PR DESCRIPTION
**Pull request by howardjohn**
**Please provide a description of this PR:**

### CODE REVIEW

File: tests/integration/security/fuzz/fuzz_test.go [hunk 1]
```diff
@@ -104,13 +104,8 @@ func deploy(t framework.TestContext, name, ns, yaml string) {
 func waitService(t framework.TestContext, name, ns string) {
 	if _, _, err := kube.WaitUntilServiceEndpointsAreReady(t.Clusters().Default().Kube(), ns, name); err != nil {
 		t.Fatalf("Wait for service %s failed: %v", name, err)
-		if name == apacheServer || name == nginxServer || name == tomcatServer {
```
Comment (nmittler) [inline tests/integration/security/fuzz/fuzz_test.go:107]: This doesn't seem related to the PR?

Comment (howardjohn) [reply tests/integration/security/fuzz/fuzz_test.go:107]: after Fatalf the function exits; this would never run. new Linter picks it up. The code is a duplicate of what is on L105 anyways

**Review by ericvn — APPROVED** [review]
I had been watching for this update to happen. Missed they built the new version. Like to get rid of the non-check messages.

**Review by ramaraochavali — APPROVED** [review]

**Review by nmittler — APPROVED** [review]

**Review by lei-tang — APPROVED** [review]
